product_docs/docs/postgres_distributed_for_kubernetes/1/quickstart.mdx

```diff
@@ -54,6 +54,11 @@ contains the definition of a PGD cluster with two data groups and a global
 witness node spread across three regions. Each data group consists of two data nodes
 and a local witness node.
 
+!!! Note Availability zones 
```

Comment:
Same as previous comment. Regions have multiple AZs, so Regions and Availability Zones?